### ISSUE TITLE
Video Quality: Muted colors, low image sharpness

### ISSUE THREAD

**Issue by wesbos** [appaheadgmbh/support#31]
The quality is slightly less than screenflow. Color issues have been logged in #29 

I see this a lot with video software, usually the bitrate is not high enough to get the ultra-crisp text I'm after.

top is screenflow, bottom is Polycapture

<img width="1622" alt="Screenshot 2024-09-26 at 4 26 29 PM" src="https://github.com/user-attachments/assets/80435f66-f21f-4f1b-b77d-c16fd3dc4ea4">

**Comment by RainerZufall9393** [comment]
I suspect that the low bit rate is also the reason for the poor video quality in my example #26 @martinlexow @appaheadgmbh

**Comment by martinlexow** [comment]
I spent last week working intensively on the image quality of screen recordings and made numerous improvements: 

- Both color reproduction (including HDR on supported monitors) and 
- image sharpness should now look significantly better. 
- Transparency/alpha channels are also supported now. 

Would you like to give it another try? All improvements are included in Version 1.4 (now available on the App Store). I’d be happy to hear your feedback!

**Comment by RainerZufall9393** [comment]
Hey @martinlexow, do your updates today only affect the screen recordings so far, or do they also affect the camera recordings?

**Comment by martinlexow** [comment]
Yes, I’ve been tweaking codecs/compression and color profiles of all video recordings.

If you notice any artifacts in your recordings @RainerZufall9393 (like you mentioned before), I’d appreciate an example so I can try to reproduce the issue on my end!

**Comment by RainerZufall9393** [comment]
All right, that sounds good! I'll test it again and give you feedback shortly. @martinlexow

**Comment by RainerZufall9393** [comment]
I have just tested it again and for me the results are unchanged. @martinlexow 

I have uploaded 3 recordings via the links below where you can compare the quality between QuickTime, PolyCapture and YouTube. It's best to watch video 1 first, then 2 and then 3 (see file names).

It's specifically about the background and the color/brightness gradations. With QuickTime, the colors and the brightness blend smoothly into each other. With PolyCapture they are no longer so smooth, but “harder” (probably because of the HEVC codec or bit rate). YouTube serves as a negative example here, which shows you exactly what I mean. You can see it immediately.

The PolyCapture video has no sound because it was recorded due to the bug. But that's not important in this case.

Video 1 (QuickTime): https://dropover.cloud/0cce29
Video 2 (PolyCapture): https://dropover.cloud/947943
Video 3 (YouTube): https://dropover.cloud/c3fc9a

**Comment by martinlexow** [comment]
Thank you for your examples — now I can clearly see what you mean 👀 

The file metadata shows that QuickTime uses `ProRes 422 LT` as the codec (which surprised me, tbh). 
> Apple ProRes 422 LT is a more highly compressed codec than Apple ProRes 422, with roughly 70 percent of the data rate and 30 percent smaller file sizes. This codec is perfect for environments where storage capacity and data rate are at a premium. The target data rate is approximately 102 Mbps at 1920x1080 and 29.97 fps.

Source: [Apple](https://support.apple.com/en-gb/102207)

However, PolyCapture currently uses `H.256/HEVC` for camera recordings. 

It’s a complex topic. Each codec has its pros and cons (image quality, recording performance, file size), and this choice should probably be left up to the user. I’ll see how I can implement this as an option in one of the upcoming versions.

**Comment by RainerZufall9393** [comment]
Hey Martin, QuickTime actually uses ProRes 422, not ProRes 422 LT.

I cut the video afterwards and exported it in ProRes 422 LT so that the file size is a bit more compact.

It's true, this really is a complex topic. It's also important to find a balance somewhere between quality on the one hand and not completely excessive file sizes on the other. A selection of different codecs and bit rates or something like that would probably be best for the user in the long term.

@martinlexow

**Comment by wesbos** [comment]
Just testing this app again after a few months - most of the bugs are solved but the quality still isn't on part with screenflow.

Screenflow top, Polycapture bottom. 

<img width="1010" alt="Screenshot 2024-12-12 at 10 47 18 AM" src="https://github.com/user-attachments/assets/8579ef99-b5da-4a52-8b9b-00841f1ce523" />

**Comment by markshust** [comment]
I'm seeing something similar to Wes, Screenflow is in back, PolyCapture in front:

<img width="451" alt="Screenshot 2024-12-20 at 8 42 43 AM" src="https://github.com/user-attachments/assets/4753eaf3-2c03-4cbf-ad3a-9da880a6a411" />
